src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -84,4 +119,8 @@ public double straightTurnPower(){
     double turnPower = error * .15;
     return turnPower;
   }
+
+  public double driveStraightGyro(double desiredAngle){  
+    return  (desiredAngle - gyroscope.getGyroAngle()) * P_VALUE;
```

Comment:
Let’s just move this line to the drive straight part of arcade drive instead of having it in its own method